sycamore/docset.py

```diff
@@ -90,16 +97,23 @@ def show(
                 document.binary_representation = f"<{binary_length} bytes>".encode("utf-8")
 
             if truncate_content and document.text_representation is not None:
-                amount_truncated = len(document.text_representation) - truncate_length
-                if amount_truncated > 0:
-                    document.text_representation = (
-                        document.text_representation[:truncate_length] + f" <{amount_truncated} chars>"
-                    )
+                document.text_representation = _truncate(document.text_representation)
 
             if not show_embedding and document.embedding is not None:
                 embedding_length = len(document.embedding)
                 document.data["embedding"] = f"<{embedding_length} floats>"
 
+            if show_elements and "elements" in document.data:
+                if not show_binary:
+                    for i, e in enumerate(document.data["elements"]):
+                        if e.get("binary_representation") is not None:
+                            binary_length = len(e["binary_representation"])
+                            e["binary_representation"] = f"<{binary_length} bytes>".encode("utf-8")
+                if truncate_content:
+                    for i, e in enumerate(document.data["elements"]):
+                        if e.get("text_representation") is not None:
+                            e["text_representation"] = _truncate(e["text_representation"])
```

Comment:
I guess this works bc docsets are copied?

Reply:
Per discussion, yes, it's a little weird but it's the same as other code.